api/server.go

```diff
@@ -635,8 +701,13 @@ func NewServer(e Explorer, cm ChainManager, s Syncer) http.Handler {
 		"GET    /contracts/:id/revisions": srv.contractsIDRevisionsHandler,
 		"POST   /contracts":               srv.contractsBatchHandler,
 
-		"GET    /pubkey/:key/contracts": srv.pubkeyContractsHandler,
-		"GET    /pubkey/:key/host":      srv.pubkeyHostHandler,
+		"GET    /v2/contracts/:id":           srv.v2ContractsIDHandler,
+		"GET    /v2/contracts/:id/revisions": srv.v2ContractsIDRevisionsHandler,
+		"POST   /v2/contracts":               srv.v2ContractsBatchHandler,
+
+		"GET    /pubkey/:key/v2/contracts": srv.pubkeyV2ContractsHandler,
```

Comment:
Did we consider making the `key` a filter on whatever route you're interested in? The `v2` here makes it slightly awkward, it kind of looks like the routing scheme is inconsistent, but it is not really. The fewer routes the better and in these handlers you would be calling similar stores as you would in the v2 contracts handler.

Reply:
Do you mean only having one route?

For example we could have:
```
		"GET    /contracts/:id":           srv.contractsIDHandler,
		"GET    /contracts/:id/revisions": srv.contractsIDRevisionsHandler,
```

these routes respond with either v1 or v2 contracts depending on the ID requested. 

But then we wouldn't have a consistent return type for those routes.